opendds_cmake_module/CMakeLists.txt

```diff
@@ -15,10 +14,8 @@ if(BUILD_TESTING)
 endif()
 
 # Make this package available to other ROS-2 packages.
-ament_package(
-  CONFIG_EXTRAS "opendds_cmake_module-extras.cmake"
-)
+ament_package()
 
 # Install CMake utilities.
-install(DIRECTORY cmake DESTINATION share/${PROJECT_NAME})
+#install(DIRECTORY cmake DESTINATION share/${PROJECT_NAME})
```

Comment:
```suggestion
```
I'd just remove it instead of a comment.

Reply:
I'm not sure if the suggestion will remove the original line or removes the line from your commit. The result should be that the commented lines in your code should be removed instead.

Reply:
Done